lib/runtime/src/pipeline/network/egress/push_router.rs

```diff
@@ -62,14 +62,20 @@ where
     _phantom: PhantomData<(T, U)>,
 }
 
-// Note there's no KV router in here because we are in dynamo-runtime. The KvRouter is in
-// dynamo-llm.
-#[derive(Default, Debug, Clone)]
+#[derive(Default, Debug, Clone, Copy, PartialEq)]
 pub enum RouterMode {
     #[default]
     Random,
     RoundRobin,
```

Comment:
Minor comment, I think we usually use `RoundRobin` for benchmark, would be a better default?

Reply:
Agreed I think a RoundRobin default would be more convenient for our own purposes and for general intuitive out-of-box behavior. While investigating perf scaling of replicas, it was an important thing to check that I was doing round-robin.

Reply:
Let's change it. Send a PR. You'll want to change the default in `dynamo-run/src/flags.rs` too (and maybe `docs/guides/dynamo_run.md`, worth checking if it's mentioned)

Reply:
https://github.com/ai-dynamo/dynamo/pull/1097